e2e/tests/api-product-list.spec.ts

```
@@ -70,15 +70,20 @@ test.describe('APIProduct List Page - Display and Filters', () => {
-    const draftLabels = await page.locator('.pf-v6-c-label.pf-m-orange:has-text("Draft")').all();
-    expect(draftLabels).toHaveLength(3);
+    // Verify Published status labels (green)
+    const publishedCount = await page
+      .locator('.pf-v6-c-label.pf-m-green:has-text("Published")')
+      .count();
+    expect(publishedCount).toBeGreaterThanOrEqual(3);
+
+    // Verify Draft status labels (orange)
+    const draftCount = await page
+      .locator('.pf-v6-c-label.pf-m-orange:has-text("Draft")')
+      .count();
```

ESLint

[error] Replace `⏎······.locator('.pf-v6-c-label.pf-m-orange:has-text("Draft")')⏎······` with `.locator('.pf-v6-c-label.pf-m-orange:has-text("Draft")')`

(prettier/prettier)